Documentation/policy/intro.rst

```diff
@@ -150,4 +161,13 @@ Endpoint Selector
 The Endpoint Selector is based on the `Kubernetes LabelSelector
 <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors>`_.
 It is called Endpoint Selector because it only applies to labels associated
-with `endpoints`.
+with an `endpoints`.
+
+.. _NodeSelector:
+
+Node Selector
+-------------
+
+The Node Selector is also based on the `LabelSelector`, however rather than
+matching on labels associated with an `endpoints`, it instead applies to labels
+associated with a node in the cluster.
```

Comment:
I have a WIP change locally with almost the exact same changes for `NodeSelectors` :smile: 

Could you just add the following sentence after this paragraph?
```rst
Node Selectors can only be used in `CiliumClusterwideNetworkPolicy`.
``` 

That way I can just drop this part of my changes and don't need to base my PR on yours.

Reply:
Done.